github-runners/k8s/entrypoint.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+if [ -n "${ADDITIONAL_PACKAGES}" ]; then
+    TO_BE_INSTALLED=$(echo ${ADDITIONAL_PACKAGES} | tr "," " " )
+    echo "Installing additional packages: ${TO_BE_INSTALLED}"
+    sudo apt-get update && sudo apt-get install -y ${TO_BE_INSTALLED} && sudo apt-get clean
+fi
+
+registration_url="https://github.com/${GITHUB_OWNER}"
+token_url="https://api.github.com/orgs/${GITHUB_OWNER}/actions/runners/registration-token"
```

Comment:
Should owner really be compiled into the image? It would make more sense to have it passed as an argument